src/test/regress/expected/multi_modifications.out

```diff
@@ -586,3 +586,36 @@ SELECT * FROM multiple_hash WHERE category = '2' ORDER BY category, data;
 ----------+------
 (0 rows)
 
+-- verify interaction of default values, SERIAL, and RETURNING
+\set QUIET on
+CREATE TABLE app_analytics_events (id serial, app_id integer, name text);
+SELECT master_create_distributed_table('app_analytics_events', 'app_id', 'hash');
+ master_create_distributed_table 
+---------------------------------
+ 
+(1 row)
+
+SELECT master_create_worker_shards('app_analytics_events', 4, 1);
+ master_create_worker_shards 
+-----------------------------
+ 
+(1 row)
+
+INSERT INTO app_analytics_events VALUES (DEFAULT, 1, 'Fauxkemon Geaux') RETURNING id;
+ id 
+----
+  1
+(1 row)
+
+INSERT INTO app_analytics_events (app_id, name) VALUES (2, 'Wayz') RETURNING id;
+ id 
+----
+  2
```

Comment:
Maybe I'm batshit paranoid (yes, you're allowed to agree), but I'd prefer id and app_id not to always be the same here.